src/controllers/locales/en.json

```diff
@@ -149,17 +149,5 @@
   "failed to update": "failed to update",
   "Information updated successfully": "Information updated successfully",
   "Bus info updated successfully": "Bus info updated successfully",
-  "There was a server error": "There was a server error",
-  "Successfully got a user": "Successfully got a user",
-  "Cannot find user with given id": "Cannot find user with given id",
-  "Successfully deleted a user": "Successfully deleted a user",
-  "Role Driver is not allowed": "Role Driver is not allowed",
-  "Role Operator is not allowed": "Role Operator is not allowed",
-  "You are not allowed. Check Your password": "You are not allowed. Check Your password",
-  "You are not allowed to view the profile": "You are not allowed to view the profile",
-  "You are not authorized, check your Id": "You are not authorized, check your Id",
-  "you are not allowed to view the profile": "you are not allowed to view the profile",
-  "Not authorized. Not owner": "Not authorized. Not owner",
-  "Not authorized": "Not authorized",
-  "Not allowed to perform this action": "Not allowed to perform this action"
```

Comment:
Were all these supposed to be deleted?

Reply:
not to be deleted, if so it was by intentionally i can fix  them